source/clear-linux/guides/maintenance/kernel-modules.rst

```diff
@@ -1,76 +1,104 @@
 security practices, see the :ref:`security` page.
 
-|CL| comes with many upstream kernel modules available for use.  If 
-you require a kernel module, be sure to check whether it is already available in |CL| first. 
+You can search for kernel module file names, which end with the :file:`.ko`
+file extension, using the :command:`swupd search` command. For example:
+:command:`sudo swupd search ${module_name}.ko`. See :ref:`swupd-search` for
+more information. 
 
-You can search for kernel module file names, which end with the :file:`.ko` 
-file extension, using the :command:`swupd search` command. For example: 
-:command:`sudo swupd search ${module_name}.ko`.
-See :ref:`swupd-search` for more information. 
 
 Request the module be added to |CL|
-===================================
+-----------------------------------
 
-If the kernel module you need is already open source 
-(e.g. in the Linux upstream) and likely to be useful to others, 
-consider submitting a request to add or enable in the |CL| kernel.
+If the kernel module you need is already open source (e.g. in the Linux
+upstream) and likely to be useful to others, consider submitting a request to
+add or enable in the |CL| kernel.
 
 Make enhancement requests to the |CL| distribution `on GitHub`_ .
 
-Build and load an out-of-tree module
-====================================
+.. _kernel-modules-availability-end:
+
+
+Build, install, and load an out-of-tree module
+==============================================
+
+In some cases you may need an out-of-tree kernel module that is not available
+through |CL|.
 
-In some cases you may need an out-of-tree kernel module that is not 
-available through |CL|.
+
+Prerequisites 
+-------------
 
 You can build and load out-of-tree kernel modules, however you must:
 
-* disable secure boot
-* disable kernel module integrity checking
-* build the module against new versions of the Linux kernel
+* Disable Secure Boot.
+* Disable kernel module integrity checking.
+* Have a kernel module package in the form of source code.
+* Rebuild the module against new versions of the Linux kernel.
 
 .. note::
 
    Any time the kernel is upgraded on your Clear Linux system, you will 
    need to rebuild your out-of-tree modules.
 
-This approach works well for individual development or testing. 
-For a more scalable and customizable approach, consider using the 
-`mixer tool`_ to provide a custom kernel and updates.
+This approach works well for individual development or testing. For a more
+scalable and customizable approach, consider using the `mixer tool`_ to
+provide a custom kernel and updates.
+
 
-Build kernel module
--------------------
+Build and install kernel module
+-------------------------------
 
 #. From a |CL| system, ensure you are running the *native* kernel. 
-   Currently only the native kernel is enabled to build and load
-   out-of-tree modules.
+   Currently only the native kernel is enabled to build and load out-of-tree
```

Comment:
Again, we now also offer -dev bundles for lts2018, lts2017, and lts (which is actually lts2018) kernels.